buddyboss/buddyboss-platform
# PROD-752 A Warning shows when gavatar is enabled and user trying to update the profile - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2869  .

### How to test the changes in this Pull Request:

1. Enable gavatar for profile
2. Login as a user who's profile picture is coming from gavatar.com
3. Try to update the profile from frontend.
4. See the issue is not happening anymore

### Proof Screenshots or Video

https://www.loom.com/share/6da954ad322f43baaf364d1448456d8e

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
